gamestonk_terminal/config_plot.py

```diff
@@ -0,0 +1,9 @@
+SCREEN_HEIGHT = 0
+SCREEN_WIDTH = 0
+
+PLOT_HEIGHT_PRECENTAGE = 100
+PLOT_WIDTH_PRECENTAGE = 50
+
+PLOT_HEIGHT = 800
+PLOT_WIDTH = 800
```

Comment:
Also, the defaults look like this to me:

USE_PLOT_AUTOSCALING = True
<img width="1280" alt="Captura de ecrã 2021-03-16, às 19 22 03" src="https://user-images.githubusercontent.com/25267873/111368426-c6dfd280-868d-11eb-86df-e2a1da953898.png">

USE_PLOT_AUTOSCALING = False
<img width="1280" alt="Captura de ecrã 2021-03-16, às 19 19 29" src="https://user-images.githubusercontent.com/25267873/111368412-c2b3b500-868d-11eb-87a3-0b68e5e957e5.png">

May I suggest these:
```
PLOT_HEIGHT_PRECENTAGE = 50
PLOT_WIDTH_PRECENTAGE = 70

PLOT_HEIGHT = 500
PLOT_WIDTH = 800
```

Reply:
Also can you add a comment above these parameters that:
PLOT_HEIGHT_PRECENTAGE
PLOT_WIDTH_PRECENTAGE
are used when AUTO_PLOT is set to True,

And that the others are used when AUTO_PLOT is False.

Reply:
Precentage is mispelled, it should be "Percentage"

Reply:
Thanks for the reply. I'll get right to it:)
I really appreciate the feedback, as I'm new to coding in teams, and got no formal / educational coding background.

Reply:
For someone with no coding background you're doing amazing!! I'll wait for your changes to merge this :) Thank you!